pages/docs/index.vue

```diff
@@ -0,0 +1,111 @@
+<template>
```

Comment:
why does the docs page take you to the first set of documentation by default?  shouldn't it actually send you to a listing of all the documentation so you can choose which one you want to open instead of scrolling down what will probably eventually be a lengthy list on the side?

Reply:
this'd also fix the problem of this file basically being a clone of the sections one and not DRY at all